### DISCUSSION TITLE
ObsidianPasteImg under wsl

### DISCUSSION THREAD

**Discussion by grezzogg** [epwalsh/obsidian.nvim#513] [Q&A] [answered]
As many of us are, I'm forced to use windows at work and wsl has been a save but one thing I'm missing quite badly is the ability to use :ObsidianPasteImg from windows terminal or even neovide.

**Comment by CODEGOAT007** [comment, marked as answer]
Little late, but this should help: https://github.com/unclejimao/WSL-Image-Clipboard-Helper

**Comment by grezzogg** [comment]
That looks promising, will try for sure.

Since i use nvim basically just for note taking I resorted to use nvim natively in windows with neovide and that worked for pasting images.
